cmd/ipfs-cluster-ctl/main.go

```diff
@@ -338,6 +338,10 @@ content.
 					Usage: "Hash function to use. Implies cid-version=1",
 					Value: defaultAddParams.HashFun,
 				},
+				cli.BoolFlag{
+					Name:  "local",
+					Usage: "Puts blocks only in local IPFS peer and then pin as per PinOptions",
```

Comment:
It's not clear what `PinOptions` is here. Better `Add to local peer but pin normally`.

And then update the related paragraph in the long description:

```
Once the adding process is finished, the content is fully added to all
allocations and pinned in them. This makes cluster add slower than a local
ipfs add, but the result is a fully replicated CID on completion.
If you prefer faster adding, use the --local flag to add directly to the local IPFS node
and pin in the destinations after that. Note that the local IPFS node may not be
among the destinations, which will leave the unpinned content in it.
```